tests/dbshell/test_mongodb.py

```diff
@@ -0,0 +1,175 @@
+import os
+import signal
+import subprocess
+import sys
+from pathlib import Path
+from unittest import mock
+
+from django_mongodb.client import DatabaseClient
+
+from django.db import connection
+from django.test import SimpleTestCase
+
+
+class MongoDBDbshellCommandTestCase(SimpleTestCase):
```

Comment:
I guess you copied this from another example but "CommandTestCase" -> "Tests". The name "TestCase" implies its meant to be inherited from. Some Django tests are a little sloppy about this.

Reply:
Changed to MongoDBDbshellCommandTests